models/messages.js

```diff
@@ -0,0 +1,21 @@
+import mongoose from 'mongoose';
+const { Schema } = mongoose;
+
+const messagesSchema = new Schema(
+	{
+		message: {
+			type: String,
+			required: true,
+		},
+		sender: {
+			type: Schema.Types.ObjectId,
+			required: true,
+		},
+	},
+	{
+		timestamps: true,
+	}
+);
+
+// Prevent recompilation of model if it already exists
+export default mongoose.models.Messages || mongoose.model('Messages', messagesSchema);
```

Comment:
Mongo auto changes this to lowercase so some of the find operations won't work. go ahead and make all the mongoose.model names lowercase